install.sh

```diff
@@ -94,11 +94,30 @@ nvm_download() {
 install_nvm_from_git() {
   local INSTALL_DIR
   INSTALL_DIR="$(nvm_install_dir)"
+  local NVM_VERSION
+  NVM_VERSION="${NVM_INSTALL_VERSION:-$(nvm_latest_version)}"
+  local FETCH_VERSION
+  FETCH_VERSION="refs/tags/$NVM_VERSION"
+  if [ -n "${NVM_INSTALL_VERSION:-}" ]; then
+    # Check if version is a tag
+    if [ "$(command git ls-remote "$(nvm_source "git")" "refs/tags/$NVM_VERSION" | wc -l)" != "0" ]; then
+      :
+    # Check if version is a ref
+    elif [ "$(command git ls-remote "$(nvm_source "git")" "$NVM_VERSION" | wc -l)" != "0" ]; then
+      :
+    # Check if version is an existing changeset
+    elif ! nvm_download -o /dev/null "$(nvm_source "script-nvm-exec")"; then
```

Comment:
This check here based on downloading nvm-exec with a given changeset seems to fail while on my other PR for Windows it seems to have worked, otherwise the CI would not pass. I need to check deeper.